.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,84 @@
+name: Trigger image scanning on pull request and changes to pr
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
```

Comment:
Can we ensure that only PRs to `main` are tested?

Reply:
done